.github/workflows/style.yaml

```diff
@@ -4,6 +4,9 @@ on:
   pull_request:
     branches: ['main']
 
+env:
+  GO_VERSION: '1.17'
```

Comment:
I'd kind of prefer to just inline these, instead of using envs to share it.

Reply:
reverted, thanks for the review and feedback